Tests/Pcap++Test/Tests/IPFragmentationTests.cpp

```diff
@@ -1073,3 +1074,30 @@ PTF_TEST_CASE(TestIPFragWithPadding)
 	delete result;
 	delete[] buffer;
 }  // TestIPFragWithPadding
+
+PTF_TEST_CASE(TestIPv4MalformedFragment)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	int bufferLength = 0;
+	uint8_t* buffer = readFileIntoBuffer("PcapExamples/ip4_bad_fragment.txt", bufferLength);
```

Comment:
I believe you also need to push the file `"PcapExamples/ip4_bad_fragment.txt"`?

Maybe instead you can push a pcap file that contains this packet?

Since `*.pcap` is in `.gitignore` you need to use the `-f` flag:
```sh
git add -f Tests/Pcap++Test/PcapExamples/ip4_bad_fragment.pcap
git commit -m "Add pcap file"
```

If you add a pcap file you can use `pcpp::PcapFileReaderDevice` to read it, like we do here: https://github.com/seladb/PcapPlusPlus/blob/51c9af39410c5eb899599357b439959cff00e299/Tests/Pcap%2B%2BTest/Tests/IPFragmentationTests.cpp#L82